temporalio/client.py

```diff
@@ -73,7 +72,7 @@ async def connect(
         ] = None,
         tls: Union[bool, TLSConfig] = False,
         retry_config: Optional[RetryConfig] = None,
-        static_headers: Mapping[str, str] = {},
+        rpc_metadata: Mapping[str, str] = {},
```

Comment:
I think it makes sense to call this just "headers" still, it's probably a more familiar term for a lot of users.

Reply:
Can't do that as we also have Temporal headers and this is a high-level API. It is important to know this high-level construct maps to low-level RPC metadata. Note, in the low-level calls I do remove the `rpc_` prefix. As for "headers" vs "metadata", this matches TypeScript and makes sense from a gRPC perspective.